pkg/workspace/library.go

```diff
@@ -123,6 +123,7 @@ func (l *Library) FindFile(path string) (*files.File, error) {
 type FileInLibrary struct {
 	File    *files.File
 	Library *Library
+	Path    string
```

Comment:
Only add a `parentLibraries []*Library` (private) and make a `fileInLib.RelativePath()` string function to use it. Populate in `ListAccessibleFiles`